### PR TITLE
Added Argument examples in openInbox() usage README

### PR DESCRIPTION
**Pull request by switch13**
Added some argument examples per conversation with @tschoffelen

### CODE REVIEW

**Comment by tschoffelen** [comment]
Thanks! I think it might be better however to do that as a separate example, since most people won't feel the need to edit any params, for which the basic example would be sufficient.

So maybe somewhere below the list of arguments?

File: README.md [hunk 1]
```diff
@@ -200,6 +211,18 @@ Email's body.
 | -------- | -------- | -------- |
 | string   | No       | null     |
 
+#### Example
+
+```javascript
+import { openComposer } from 'react-native-email-link'
+
+openInbox({
```
Comment (tschoffelen) [inline README.md]: I think this should be `openComposer`

Comment (switch13) [reply README.md]: sorry about that! nice catch!

**Comment by tschoffelen** [comment]
Thank you!

**Comment by switch13** [comment]
> Thank you!

thank you! this is a cool package!

**Pull request merged**